xrally_kubernetes/service.py

```diff
@@ -1117,6 +1119,157 @@ def delete_statefulset(self, name, namespace, status_wait=True):
                                    resource_type="StatefulSet",
                                    namespace=namespace)
 
+    @atomic.action_timer("kubernetes.list_node")
+    def list_node(self):
```

Comment:
What does this mean? get a single node or listing nodes?

Reply:
second. Will fix.